src/funding/paylater/config.jsx

```diff
@@ -20,6 +20,21 @@ export function getPaylaterConfig() : FundingSourceConfig {
             BUTTON_LAYOUT.VERTICAL
         ],
 
+        eligible: ({ experiment, fundingEligibility, fundingSource }) => {
+            const { paylater } = fundingEligibility;
+            if (
```

Comment:
Minor nitpick but for readability sake, and if code is not in hotpath, I sometimes will make some functions such as `hasTaggedEligibility` or `hasStandaloneButtons` to make these conditionals a tad easier to glance over. The comments you added are great though so thanks for those! Just sharing an alternative we might explore later. No change required

Reply:
Hi, that's definitely a good point. One thing I'm trying to keep in mind is that this code is shipped to the client so anything to minimize the bundle size would be ideal. I think if we don't absolutely need a helper it would be best to avoid it. But maybe my understanding of how additional helper functions impact build size is wrong

Reply:
In my opinion, I think readability should be a major focus as should keeping code tight. A small number of extra lines is worth understanding/maintainability to me. I am, however, certainly glad its on your mind to keep our bundle sizes in check! I'm gonna mark this as resolved as I think it's an out of scope conversation for the changeset, but my intent was to get this on folks minds as a discussion and thank you so much for sharing thoughts with me! Happy to continue a discussion in the future!